ui/react-components/components/AllAppointmentServices/AllAppointmentServices.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import './AllAppointmentServices.scss'
```

Comment:
Prefer to use css modules and classnames. That generates unique classnames after compilation which prevents from conflicting with similar class names in other modules/components.

Reply:
looking into it

Reply:
done